Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1411,10 +1411,7 @@
 
 ; Needed for [CAContext remoteContextWithOptions]
 #if HAVE(SANDBOX_STATE_FLAGS)
-(with-filter
-    (require-any
-        (require-not (lockdown-mode))
-        (require-not (state-flag "EnableExperimentalSandbox")))
+(with-filter (require-not (lockdown-mode))
```

Comment:
What's the significance of the lockdown mode clause here?

Reply:
Due to PDFs, we will only be able to block CARenderServer on macOS when LDM is enabled.

Thanks for reviewing!